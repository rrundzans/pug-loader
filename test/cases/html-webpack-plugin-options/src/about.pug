.title= htmlWebpackPlugin.options.title
.lang= htmlWebpackPlugin.options.tmplData.lang
.description= dataFromQuery.options.description
img(src=require('./image.jpeg'))