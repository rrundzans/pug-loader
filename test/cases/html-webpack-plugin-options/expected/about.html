<div class="title">About</div><div class="lang">en-US</div><div class="description">Custom about page</div><img src="assets/images/image.f78b30f4.jpeg">